tutorial/sequence/Lesson01/acquire_102.md

```diff
@@ -25,15 +25,25 @@ You will also need a programmer's plain text editor, XML/XSLT editor or IDE (int
 
 Same as [Setup 101](acquire_101.md).
 
+Also see [the set of XProc worksheets](../../worksheets/) collected so far for this tutorial.
+
+## Step Zero: XProc steps
```

Comment:
This new step zero is great! It clarifies things that were so mysterious to me when I looked at XProc code before learning the mental model.